### ISSUE TITLE
Multiple notes selection instead of note switching 

### ISSUE THREAD

**Issue by dec0dOS** [standardnotes/web#679]
Hello, Standard Notes team!

Under some special conditions, instead of note switching, you're getting the note you want to switch and the previous one selected.

I've faced that issue some time ago and could not reproduce it for a long time. That issue drives me crazy. Today, I've tried to reproduce it in different cases. It is worth mentioning that this issue has been introduced a few months ago: the 3.7.1 release is just fine.

Steps to reproduce:
1. Open SN
2. Open any note
3. Press the CMD+SHIFT+4, then ESC (or press the CMD+H, then open the app again) 
4. Try to open another note. Instead, the previous note and another one are selected.

It could be reproduced in the https://app.standardnotes.org in Safari, and in macOS Standard Notes App. I could not reproduce this bug with Firefox on PopOS. 

I've tried to create the detailed screen capture, but the bug appears instantly when I'm attempting to start the screen capture. So I could provide only the short one.
 
https://user-images.githubusercontent.com/14824142/137122644-06a1bf0d-69fa-47c5-9c46-5471964c5957.mov

The comment to the screen capture: I didn't hold any key when was trying trying to select another note.

I suppose that this issue is connected with this Reddit thread https://www.reddit.com/r/StandardNotes/comments/p3l3bj/multiple_notes_selected/

**Comment by JaspalSuri** [comment]
Hi @dec0dOS, thank you for the report; we'll look into it.

**Comment by JaspalSuri** [comment]
Hi @dec0dOS, not at the moment, but it's still on our list of issues to fix.

**Comment by dec0dOS** [comment]
Hey, seems that the issue has been resolved. I've removed the old quick tags extension, it has been the source of the problem.

**Comment by dec0dOS** [comment]
Could you reproduce the bug? I've faced the same bug even after upgrading to the new mac with the latest macOS.

**Comment by fixje** [comment]
seems to be related to or a duplicate of #607

I'm seeing the issue occasionally, but haven't found a deterministic way to reproduce

**Comment by dec0dOS** [comment]
This issue makes the SN almost unusable. Please fix this ASAP.

**Comment by moughxyz** [comment]
This happens in macOS when you take a screenshot because surprisingly the OS doesn't tell us when you've "unfocused" from our app. In the latest version there is at least a workaround in that if you press command again and release, it will reset selection mode and go back to single selection mode. We haven't been able to find a solution to the actual screenshot problem.

**Comment by dec0dOS** [comment]
The same problem persists even with the CMD+H shortcut, so it makes SN almost unusable on macOS. This is one of the main multitask shortcuts. Please make an option to disable multiple notes selection if there is no way to fix it.

**Comment by moughxyz** [comment]
Try 3.9.20. I don't think 3.9.13 had the fix.

**Comment by moughxyz** [comment]
Right, but at the very least with 3.9.20 pressing cmd again then releasing should reset the behavior and disable multiple selection mode.

**Comment by dec0dOS** [comment]
Is there any other option besides maintaining the fork without multiple selection feature?
